monai/networks/utils.py

```diff
@@ -17,8 +17,32 @@
 import torch
 import torch.nn.functional as f
 
+def one_hot(labels, num_classes, dtype=torch.float):
+    """
+    For a tensor `labels` of dimensions B1[spatial_dims], return a tensor of dimensions `BN[spatial_dims]`
+    for `num_classes` N number of classes.
+
+    Example:
+
+        For every value v = labels[b,1,h,w], the value in the result at [b,v,h,w] will be 1 and all others 0.
+        Note that this will include the background label, thus a binary mask should be treated as having 2 classes.
+    """
+    assert labels.dim() > 1, "labels should have dim of 2 or more."
+
+    sh = list(labels.shape)
+    assert sh[1] == 1, "labels should have a channel with length equals to one."
+
+    sh[1] = num_classes
+    o = torch.zeros(size=sh, dtype=dtype, device=labels.device)
+    labels = o.scatter_(dim=1, index=labels.long(), value=1)
+
+    ## it's contiguous by construction, no need to check
```

Comment:
Please remove the comments if no use anymore.

Reply:
done